content/providers/01-ai-sdk-providers/02-openai.mdx

```diff
@@ -804,3 +804,62 @@ const model = openai.image('dall-e-3');
 | ---------- | ------------------------------- |
 | `dall-e-3` | 1024x1024, 1792x1024, 1024x1792 |
 | `dall-e-2` | 256x256, 512x512, 1024x1024     |
+
+## Transcription Models
+
+You can create models that call the [OpenAI transcription API](https://platform.openai.com/docs/api-reference/audio/transcribe)
+using the `.transcription()` factory method.
+
+The first argument is the model id e.g. `whisper-1`.
+
+```ts
+const model = openai.transcription('whisper-1');
+```
+
+OpenAI transcription models support a single `audio` argument which can be a `DataContent` (`string | Uint8Array<ArrayBufferLike> | ArrayBuffer | Buffer<ArrayBufferLike>`) or a `URL`.
+
+```ts
+const model = openai.transcription('whisper-1', {
+  audio: new File([], 'audio.mp3', { type: 'audio/mp3' }),
+});
+```
+
+You can also pass additional provider-specific options using the `providerOptions` argument. For example, supplying the input language in ISO-639-1 (e.g. `en`) format will improve accuracy and latency.
+
+```ts
+const model = openai.transcription('whisper-1', {
+  providerOptions: { openai: { language: 'en' } },
+});
```

Comment:
same here - provider options should be on the request and have a pass through mechanism similar to `generateText`

Reply:
still seems like the wrong location for `providerOptions` - would need to be passed into a full `transcribe` call not the model contructor